compiler/src/iree/compiler/Codegen/LLVMGPU/KernelConfig.cpp

```diff
@@ -115,6 +115,17 @@ bool isROCmBackend(IREE::GPU::TargetAttr target) {
   return target.getArch().starts_with("gfx");
 }
 
+static bool needsLoweringConfigPropagation(
+    IREE::Codegen::DispatchLoweringPassPipeline pipeline) {
+  using Pipeline = IREE::Codegen::DispatchLoweringPassPipeline;
+  SmallVector<Pipeline> supportedPipelines = {
```

Comment:
nit: Add a short comment that these are the pipelines that do not need propagation (at first glance it looks like the opposite based on the helper name)